docker-compose.yaml

```diff
@@ -10,6 +10,8 @@ services:
       APPLICATIONINSIGHTS_CONNECTION_STRING: ${APPLICATIONINSIGHTS_CONNECTION_STRING}
       DDI_API_BASE_URL: ${DDI_API_BASE_URL:-test}
       DDI_EVENTS_BASE_URL: ${DDI_EVENTS_BASE_URL:-test}
+      OS_PLACES_API_KEY: ${OS_PLACES_API_KEY:-test}
+      OS_PLACES_API_BASE_URL: ${OS_PLACES_API_BASE_URL:-test}
       COOKIE_PASSWORD: who-likes-cookies-i-like-cookies-everybody-likes-cookies
```

Comment:
Do we need this in docker-compose.test as well?

Reply:
can the defaults reflect the ones in jest.setup.single?  if they are different then the tests error when you running them locally?